src/accordion/accordion.ts

```diff
@@ -76,7 +82,7 @@ export interface NgbPanelChangeEvent {
   template: `
   <div class="card">
     <template ngFor let-panel [ngForOf]="_panels">
-      <div class="card-header" [class.active]="_isOpen(panel.id)">
+      <div [class]="'card-header card-' + (panel.type ? panel.type: type)" [class.active]="_isOpen(panel.id)">
```

Comment:
You need to check if type is set and only then add `card-type` class. We are doing sth similar in progress bar: https://github.com/ng-bootstrap/ng-bootstrap/blob/master/src/progressbar/progressbar.ts#L11